README.md

```diff
@@ -7,3 +7,46 @@
 [![RubyGem Downloads](https://img.shields.io/gem/dt/beaker-hcloud.svg)](https://rubygems.org/gems/beaker-hcloud)
 
 A [beaker](https://github.com/voxpupuli/beaker) extension for provision Hetzner Cloud instances.
+
+## Installation
+
+Include this gem alongside Beaker in your Gemfile or project.gemspec. E.g.
+
+```ruby
+# Gemfile
+gem 'beaker', '~>4.0'
+gem 'beaker-hcloud'
+# project.gemspec
+s.add_runtime_dependency 'beaker', '~>4.0'
+s.add_runtime_dependency 'beaker-hcloud'
+```
+
+## Authentication
+
+You need to create an API token using Hetzner's cloud console. Make
+sure to create the token in the correct project.
+
+`beaker-hcloud` expects the token to be in the `HCLOUD_TOKEN`
```

Comment:
@ekohl when we introduce a new environment variable, do we want it prefixed with `BEAKER_`?

Reply:
Sounds like a good idea to me.

Reply:
Sorry for the delay, but I finally got around to renaming the environment variable.